stubs/vms/4db71538-9255-4ba3-a291-1240682dd8c6/content.json

```diff
@@ -0,0 +1,341 @@
+      "id" : "00000000-0000-0000-0000-000000000000"
+    },
+    "watchdogs" : { },
+    "actions" : {
+      "link" : [ {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/detach",
+        "rel" : "detach"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/shutdown",
+        "rel" : "shutdown"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/start",
+        "rel" : "start"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/stop",
+        "rel" : "stop"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/suspend",
+        "rel" : "suspend"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/reset",
+        "rel" : "reset"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/ticket",
+        "rel" : "ticket"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/migrate",
+        "rel" : "migrate"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/reboot",
+        "rel" : "reboot"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/commitsnapshot",
+        "rel" : "commitsnapshot"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/clone",
+        "rel" : "clone"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/freezefilesystems",
+        "rel" : "freezefilesystems"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/logon",
+        "rel" : "logon"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/maintenance",
+        "rel" : "maintenance"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/previewsnapshot",
+        "rel" : "previewsnapshot"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/autopincpuandnumanodes",
+        "rel" : "autopincpuandnumanodes"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/reordermacaddresses",
+        "rel" : "reordermacaddresses"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/thawfilesystems",
+        "rel" : "thawfilesystems"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/undosnapshot",
+        "rel" : "undosnapshot"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/cancelmigration",
+        "rel" : "cancelmigration"
+      }, {
+        "href" : "/ovirt-engine/api/vms/4db71538-9255-4ba3-a291-1240682dd8c6/export",
+        "rel" : "export"
+      } ]
+    },
+    "name" : "vm-for-migration",
```

Comment:
we don't support two vms with the same name within the same cluster in ovirt so let's give it a different name (I guess we'll fail to import both to the same namespace in k8s)

Reply:
ack, changed to `vm-for-insecure-migration`